ARMmbed/mbedtls
# Add partial support to import/copy opaque keys

## Description
Add support to size and import opaque keys.
Extend test driver support to copy opaque keys.  

## Status
**READY**

## Requires Backporting
NO  


## Migrations
NO

## Additional comments

## Todos


## Steps to test or reproduce
import/ copy of keys to an external location would currently fail as the name indicates, and with this improvement, it should no longer be a create fail.
_Create failure: unsupported location for a volatile key
depends_on:MBEDTLS_PSA_CRYPTO_STORAGE_C
create_fail:PSA_KEY_LIFETIME_FROM_PERSISTENCE_AND_LOCATION(PSA_KEY_PERSISTENCE_VOLATILE, PSA_CRYPTO_TEST_DRIVER_LOCATION):0:PSA_ERROR_INVALID_ARGUMENT_
